### PR TITLE
Fix/providers without signer accounts

### PR DESCRIPTION
**Pull request by livingrock7**

### CODE REVIEW

File: src/ERC20ForwarderClient.js [hunk 1]
```diff
@@ -312,15 +312,17 @@ class ERC20ForwarderClient {
...
-        );
-
       await this.checkTokenSupport(token);
 
-      if (!userAddress)
-        userAddress = await this.provider.getSigner().getAddress();
+      if (!userAddress) {
+        if (!this.isSignerWithAccounts) {
+          throw new Error(
+            "Provider object passed to Biconomy does neighter have user account information nor userAddress is passed. Refer to docs or contact Biconomy team to know how to use ERC20ForwarderClient properly"
```
Comment (tomarsachin2271) [inline src/ERC20ForwarderClient.js:320]: typo in neighter 
Provider object passed to Biconomy neither has user account information not userAddress is passed. Refer to docs or contact Biconomy team to know how to use ERC20ForwarderClient properly.

Comment (livingrock7) [reply src/ERC20ForwarderClient.js:320]: fixed

Comment (livingrock7) [reply src/ERC20ForwarderClient.js:320]: fixed

Comment (livingrock7) [reply src/ERC20ForwarderClient.js:320]: fixed

**Pull request merged**